views/partials/search-form-fields.html

```diff
@@ -183,7 +183,7 @@
   <div class="controls">
     <button type="submit"><span>Search</span></button>
     {% if advancedSearchIsVisible %}
```

Comment:
The container is hidden by default, but the user can click on "Advanced Search" and see it. Maybe it's overkill, and it's OK to show the "Clear" button even if there's nothing to clear, but it doesn't hurt considering it's already here.